Enterprise-CMCS/eAPD
# Dmirano/4570 new prev activities mmis

Resolves #4570 

### Description

Updates the Results of Previous Activities page for MMIS. This adds DDI and M&O funding types.

### Significant changes or possible side effects

Created a brand new file vs adapting the current HITECH one. 
Also changed the selector to not be specific to ffp level. (6 functions re-factored into 2)

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| --- | --- | --- | -------- |
| Results of Previous Activities page  | Changing value (90 DDI, 75 DDI/M&O, 50 DDI/M&O) | Correct function gets called with the correct values | Jest |
| Results of Previous Activities page  | Calculating the totals for the totals table | Correct values are calculated | Jest |
| Results of Previous Activities page  | On the page | All tables are present | Cypress |


### Steps to manually verify this change

1. Navigate the the Results of Previous Activities page for an MMIS APD
2. Verify design matches Figma
3. Verify Results of Previous Activities page isn't changed for HITECH APDs
4. (dev) Please check that I did my mongo migrations correctly 😅 

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
